From d6b2df1829c772cbcfa16d6acb6363750315d3e6 Mon Sep 17 00:00:00 2001
From: Mat Kowalski <mko@redhat.com>
Date: Thu, 9 Jul 2026 11:07:30 +0200
Subject: [PATCH 25/26] MCO: name the mirror pod for the frr-status exporter

frr-status fetches its own pod object; static pods must pass the
mirror pod name explicitly.
---
 templates/master/00-master/on-prem/files/0000-frr-k8s.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/templates/master/00-master/on-prem/files/0000-frr-k8s.yaml b/templates/master/00-master/on-prem/files/0000-frr-k8s.yaml
index 2533098c6..351de8455 100644
--- a/templates/master/00-master/on-prem/files/0000-frr-k8s.yaml
+++ b/templates/master/00-master/on-prem/files/0000-frr-k8s.yaml
@@ -204,6 +204,7 @@ contents:
         args:
         - --node-name=$(NODE_NAME)
         - --namespace=openshift-frr-k8s
+        - --pod-name=frr-k8s-$(NODE_NAME)
         env:
         - name: NODE_NAME
           valueFrom:
